RuGenerous/rugenerous-frontend
# Feature/redemption gtfo

## Description
Add Redemption Page
Add Styling for Redemption Page
Add the Smart Contract into to the config
Create a thunk that checks the RUG token has allowance on aforementioned smart contract, allows you to approve if not, and allows you to stake if true

## How To Review This

- Add a small amount of USDC.e into smart contract - 0xBeBD9b389281cdDbdd29BBcAe7d0Ab943996653f.
- Acquire RUG token
- Approve contract on UI - Redeem page
- Refresh the dApp
- Then swap the acquired RUG - Ensure the amount of USDC received from the swap reflects the applied RFV * amount of RUG

## Checklist:
